webapp/channels/src/components/channel_settings_modal/channel_settings_modal.test.tsx

```diff
@@ -229,4 +234,60 @@ describe('ChannelSettingsModal', () => {
         // The archive tab should not be in the document
         expect(screen.queryByRole('tab', {name: 'archive'})).not.toBeInTheDocument();
     });
+
+    it('should not show configuration tab with no license', async () => {
+        const baseState: DeepPartial<GlobalState> = {
+            entities: {
+                general: {
+                    license: {
+                        SkuShortName: '',
+                    },
+                },
+            },
+        };
+        renderWithContext(<ChannelSettingsModal {...baseProps}/>, baseState as GlobalState);
+        expect(screen.queryByTestId('configuration-tab-button')).not.toBeInTheDocument();
+    });
+
+    it('should not show configuration tab with professional license', async () => {
+        const baseState: DeepPartial<GlobalState> = {
+            entities: {
+                general: {
+                    license: {
+                        SkuShortName: 'professional',
+                    },
+                },
+            },
+        };
+        renderWithContext(<ChannelSettingsModal {...baseProps}/>, baseState as GlobalState);
+        expect(screen.queryByTestId('configuration-tab-button')).not.toBeInTheDocument();
+    });
+
+    it('should not show configuration tab with enterprise license', async () => {
+        const baseState: DeepPartial<GlobalState> = {
+            entities: {
+                general: {
+                    license: {
+                        SkuShortName: 'enterprise',
+                    },
+                },
+            },
+        };
+        renderWithContext(<ChannelSettingsModal {...baseProps}/>, baseState as GlobalState);
+        expect(screen.queryByTestId('configuration-tab-button')).not.toBeInTheDocument();
+    });
+
+    it('should show configuration tab when premium license', async () => {
+        const baseState: DeepPartial<GlobalState> = {
+            entities: {
+                general: {
+                    license: {
+                        SkuShortName: 'premium',
```

Comment:
quesiton: didn't this license name changed recently? if so, for future clarity, even in the tests should be updated.

Reply:
The task for renamining the license will be done by me two weeks from now once I'm back from PTO. Until then, we should use the "premium" SKU name so as to ensure everything is renamed correctly.